salt/salt.spec

```diff
@@ -506,6 +506,8 @@ Patch219:       fix-salt-ssh-opts-poisoning-bsc-1197637-3002.2-500.patch
 Patch220:       prevent-affection-of-ssh.opts-with-lazyloader-bsc-11.patch
 # PATCH-FIX_OPENSUSE: https://github.com/openSUSE/salt/pull/504
 Patch221:       fix-regression-with-depending-client.ssh-on-psutil-b.patch
+# PATH-FIX_OPENSUSE: https://github.com/openSUSE/salt/pull/511
```

Comment:
If there is an upstream patch for the fix, it's better to specify the URL pointing to it with `PATCH-FIX_UPSTREAM`

Reply:
Thank you Victor. In this case an upstream patch was not needed.

Reply:
I mean not the PR to upstream created by you, but the PR fixing the same issue in upstream, maybe slightly different way. We just need to keep tracking if there upstream fix for the issue.